scripts/cl_logger.lua

```diff
@@ -39,11 +39,11 @@ local function processDeath(ped)
         end
     else
         if IsEntityAPed(killerPed) and IsPedAPlayer(killerPed) then
-            killer = NetworkGetPlayerIndexFromPed(killerPed)
+            killer = GetPlayerServerId(NetworkGetPlayerIndexFromPed(killerPed))
```

Comment:
would probably be cleaner to add onto the if statement on line 57 instead of adding it to each `NetworkGetPlayerIndexFromPed`
```lua
if killer == nil then
	killer = false
elseif killer then
	-- we have their player, we need their server id
	killer = GetPlayerServerId(killer)
end
```